### PR TITLE
Trials Project 1

### PR DESCRIPTION
**Pull request by jasonfelice**
### Alan Turing Timeline

- Added a main section with Alan Turing's picture
- Added a timeline section
- Styled the web page with CSS

### CODE REVIEW

**Review by jasonfelice — COMMENTED** [review]
Please make sure there are no linter errors.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+  <style>
+    body {
+      display: block;
```
Comment (jasonfelice) [inline index.html]: By default, the body is a block-level element.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+</head>
+
+<body>
+  <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script> -->
+
+  <main id="main">
+    <h1 id="title">Alan Turing</h1>
+    <p>The inventor of the first computer</p>
+    <figure id="img-div">
+      <img src="https://www.denverpost.com/wp-content/uploads/2020/02/Code_Breaker_Memorabilia_51691.jpg">
```
Comment (jasonfelice) [inline index.html:18]: It is always a good idea to add an alt attribute to the images that are part of the content.

**Review by vic778 — COMMENTED** [review]
# STATUS: INVALID!!!  :x: :x: :x: 

Hi @jasonfelice ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code, please make sure your implement the following changes before you resubmit for review. 

## Required Changes ♻️
_Check the comments below._
- [x] Please you have submitted a project that contains a linters errors, kindly fix all linters errors and submit again your PR

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 2 above)
Comment (Mre55) [reply index.html:18]: [OPTIONAL] Good job noticing the detailed technical part of your code 👍. But I would suggest you can explain the "why" a little more in this case it's more generic and vague.

File: index.html (hunk 1 above)
Comment (Mre55) [reply index.html]: [OPTIONAL] Good job mentioning about the body has to be block-level. One thing to improve here is that you can request in a polite way this comment is more like order. Try to improve that in the future.

**Review by Mre55 — APPROVED** [review]
**Hi @jasonfelice,**

# Status: APPROVED 🎊 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**